core/domain/search_services.py

```diff
@@ -27,6 +27,9 @@
 # Name for the collection search index.
 SEARCH_INDEX_COLLECTIONS = 'collections'
 
+# Name for the story search index.
+SEARCH_INDEX_STORIES = 'stories'
```

Comment:
Hm, I don't think stories are meant to be searchable -- why do we need to change this file?

Reply:
It was not needed, removed it.